python/libsharp/libsharp.pyx

```diff
@@ -4,6 +4,15 @@ __all__ = ['legendre_transform', 'legendre_roots', 'sht', 'synthesis', 'adjoint_
            'analysis', 'adjoint_analysis', 'healpix_grid', 'triangular_order', 'rectangular_order',
            'packed_real_order']
 
+# For compatibility with mpi4py <= 1.3.1
+# For newer versions, one can use the MPI._adressof function
+ctypedef object Comm
+try:
+    from mpi4py.MPI cimport Comm
```

Comment:
Hmm, the stuff about mpi4py paths into build system is probably not right. Cython should look in PYTHONPATH etc for pxd files, and no C headers are needed. I'll tick off the boxes as "done" as they are likely not needed.

So moving this to another pyx, which is only built if ENABLE_MPI, should do the trick.